src/components/ComponentDetails/PropTable/PropDescription/index.css

```diff
@@ -0,0 +1,4 @@
+.prop-description .ic-typography-subtitle-small::before {
+  color: var(--ic-status-error);
+  content: "*";
+}
```

Comment:
I'm assuming this has already been discussed but shouldn't the word required also be red to draw attention to it being required?

Reply:
In the figma file it only showed the asterisk as red, so I followed that.
<img width="614" alt="Screenshot 2023-01-20 at 11 32 06" src="https://user-images.githubusercontent.com/111882034/213685630-f0588382-7801-4ebd-ab42-a3ecab4b7d4c.png">